src/main/java/org/broadinstitute/dsm/route/ClinicalKitsRoute.java

```diff
@@ -0,0 +1,79 @@
+package org.broadinstitute.dsm.route;
+
+import lombok.NonNull;
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.ddp.db.TransactionWrapper;
+import org.broadinstitute.dsm.db.DDPInstance;
+import org.broadinstitute.dsm.model.ClinicalKit;
+import org.broadinstitute.dsm.model.ParticipantWrapper;
+import org.broadinstitute.dsm.model.bsp.BSPKitInfo;
+import org.broadinstitute.dsm.model.bsp.BSPKitQueryResult;
+import org.broadinstitute.dsm.statics.RequestParameter;
+import org.broadinstitute.dsm.util.NotificationUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Request;
+import spark.Response;
+import spark.Route;
+
+import java.sql.Connection;
+import java.util.Map;
+import java.util.Optional;
+
+public class ClinicalKitsRoute implements Route {
+
+    private static final Logger logger = LoggerFactory.getLogger(ClinicalKitsRoute.class);
+
+    private NotificationUtil notificationUtil;
+
+    public ClinicalKitsRoute(@NonNull NotificationUtil notificationUtil) {
+        this.notificationUtil = notificationUtil;
+    }
+
+    @Override
+    public Object handle(Request request, Response response) {
+        String kitLabel = request.params(RequestParameter.LABEL);
+        if (StringUtils.isBlank(kitLabel)) {
+            throw new RuntimeException("Please include a kit label as a path parameter");
+        }
+
+        return TransactionWrapper.inTransaction(conn -> {
+            return getClinicalKit(conn, kitLabel, response);
+        });
+    }
+
+    private Object getClinicalKit(Connection conn, String kitLabel, Response response) {
```

Comment:
It is always preferable to make methods more specific to keep high cohesion, as it seems, this method has only one return point and it returns object of type `ClinicalKit`, so it can be easily changed from `Object` to `ClinicalKit`, such small change would bring more understanding of method.